.github/workflows/admin-pr.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           firebase setup:emulators:ui
 
       - name: Install Dependencies
-        run: yarn install
+        run: npm install
 
       - name: Run Tests
         run: yarn emulators:test
```

Comment:
Shall we completely switch for all commands to npm?

Reply:
Yes, I would switch everywhere. I updated all files now.